### ISSUE TITLE
Update to support multiple antibody levels

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/inc2prev#13]
Data now available with multiple antibody levels: https://twitter.com/theosanderson/status/1481675533524127748

This enables the option of some more complex modelling and better handle of multiple rounds of vaccination etc. Needs some thought to keep the approach general and what exactly could be included

**Comment by sbfnk** [comment]
Yes, also I think N-gene antibodies are in the works by ONS, which would give antibody levels from infection only.

**Comment by seabbs** [comment]
Nice!
